package.json

```diff
@@ -14,9 +14,11 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "axios": "^1.7.2",
     "discord.js": "^14.14.1",
     "dotenv": "^16.4.5",
     "jest": "^29.7.0",
+    "node-cron": "^3.0.3",
```

Comment:
[node-cron](https://www.npmjs.com/package/node-cron) is not used. I guess you are going to use it for the next part of the task but you may end up using another package to implement the next task and this one stays installed and we don't notice it, so I think it's better to remove it for now. 😼